Makefile

```diff
@@ -126,6 +126,7 @@ test-cmd: build
 # Example:
 #   make test-end-to-end
 test-end-to-end: build
+	hack/env hack/verify-generated-protobuf.sh # Test the protobuf serializations when we know Docker is available
```

Comment:
Is this wired to check the upstream ones too?  Actually, given protobuf rules, we probably need additional checks to ensure that we're compatible, right?